linkerd/app/inbound/src/lib.rs

```diff
@@ -39,8 +39,18 @@ pub struct Config {
     pub allow_discovery: NameMatch,
     pub proxy: ProxyConfig,
     pub policy: policy::Config,
-    pub profile_idle_timeout: Duration,
     pub allowed_ips: transport::AllowIps,
+
+    /// Configures how long the proxy will wait for a profile response for
+    /// skipping route information.
```

Comment:
tiny nit: this wording feels a little unclear?
```suggestion
    /// Configures how long the proxy will wait for a profile response. If no
    /// profile response is received before this timeout elapses, route information
    /// is skipped.
```
or even just
```suggestion
    /// Configures how long the proxy will wait for a profile response before
    /// skipping route information.
```